docs/staking-modules/csm/guides/addresses.md

```diff
@@ -98,4 +100,33 @@ These changes grant `managerAddress` the ultimate control over the Node Operator
 
 :::info
 If the `extendedManagerPermissions` option is used, `managerAddress` should be set to a secure address ("cold" wallet or multisig)
-:::
\ No newline at end of file
+:::
+
+## Examples
+
+### Default
+
+By default, the Node Operator's creator is set as a `managerAddress` and `rewardAddress`.
+
+![addresses-2](../../../../static/img/csm/addresses-2.png)
+
+It is recommended that addresses be re-assigned according to one of the optimal configurations below.
+
+### Optimal for solo operator
+
+It is considered optimal to set `managerAddress` to a "Hot" or easily accessible address and `rewardAddress` to a secure "Cold" wallet or even a MultiSig if you are a solo operator.
+
+![addresses-3](../../../../static/img/csm/addresses-3.png)
+
+### Optimal for DVT operator
+
+Several optimal options exist for the DVT operators.
+
+The first is to set `managerAddress` to a "Hot" or just easily accessible address controlled by the cluster coordinator and `rewardAddress` to a cluster MultiSig.
```

Comment:
"to a cluster MultiSig." => "to a cluster MultiSig. This is only recommended in cases where cluster participants trust the coordinator completely (or have other types of guarantees/agreements in place), or if the DVT setup is an intra-operator setup rather than an inter-operator setup."